modules/agar/src/main/ts/ephox/agar/mouse/Clicks.ts

```diff
@@ -85,25 +85,5 @@ const point = (type: string, button: number, element: SugarElement<Node>, x: num
   element.dom.dispatchEvent(ev);
 };
 
-export {
-  event,
-  Settings,
-  EventType,
-  leftClickButton,
-  middleClickButton,
-  rightClickButton,
-  leftClickButtons,
-  rightClickButtons,
-  middleClickButtons,
-  click,
-  mouseDown,
-  mouseUp,
-  mouseMove,
-  mouseOver,
-  mouseOut,
-  contextMenu,
-  // deprecate these
-  point,
-  trigger,
-  dblclick
-};
+export type { Settings, EventType };
+export { event, leftClickButton, middleClickButton, rightClickButton, leftClickButtons, rightClickButtons, middleClickButtons, click, mouseDown, mouseUp, mouseMove, mouseOver, mouseOut, contextMenu, point, trigger, dblclick };
```

Comment:
I've notices that the LSP organize imports some times messes up the exports. I've dealt with this by tweaking my LSP in NVIM to never do that but that is not a great option. I wonder if we should start per function/type exports over this at the end export since then that problem goes away.

For this one we should restore it into multiline.